Conch/lib/Conch/Legacy/Control/Device/Environment.pm

```diff
@@ -55,26 +47,26 @@ sub validate_cpu_temp {
 
 		my $method = "${cpu}_temp";
 
-		if ( $device_env->$method > $criteria->crit ) {
+		if ( $device_env->$method > 70 ) {
```

Comment:
All of these values have remained unchanged in the database. They will likewise be code-specified values in the new validation system.